scripts/start.sh

```diff
@@ -32,6 +32,9 @@ function setup_mysql() {
     for f in /docker-entrypoint-initdb.d/*; do
         cat $f | mysql -uroot -proot --host ${SQLFLOW_MYSQL_HOST} --port ${SQLFLOW_MYSQL_PORT}
     done
+    # Grant all privileges to any remote hosts so that the sqlserver can be scaled into more than one replicas.
```

Comment:
Maybe add tips about configuring this with care in production env.

Reply:
done.